### DISCUSSION TITLE
I'm verified for GitHub education pack but unable to get access on Azure @sh20raj

### DISCUSSION THREAD

**Discussion by SH20RAJ** [community/community#129710] [GitHub Education]
### Hi everyone,

Academic Verification
Academic verification required
The account you are signed in is not yet verified to access offer benefits. Please use the form below to verify.
Verification method

We could not confirm your status as a student. Click [here](https://go.microsoft.com/fwLink/p/?LinkID=2136428&clcid=0x409) to begin the academic verification process. This process may take up to a few days. Once verification is complete, please come back and sign in with GitHub to redeem Azure for Students.

<img width="1470" alt="Screenshot 2024-06-25 at 2 22 46 PM" src="https://github.com/community/community/assets/66713844/62499a2e-37bd-4a78-a029-dcc9cd280532">
<img width="1470" alt="Screenshot 2024-06-25 at 2 22 30 PM" src="https://github.com/community/community/assets/66713844/dee0cd75-e211-4f70-b972-57fcad560ab0">
<img width="1470" alt="Screenshot 2024-06-25 at 2 22 15 PM" src="https://github.com/community/community/assets/66713844/4a8440e5-7fc6-44e3-8301-83a7b396a926">
<img width="1470" alt="Screenshot 2024-06-25 at 2 22 07 PM" src="https://github.com/community/community/assets/66713844/0794974b-8d72-4bf3-8ed0-632d71747994">

**Comment by jericho1050** [comment]
![Screenshot 2024-06-30 at 7 10 33 PM](https://github.com/community/community/assets/118679068/85fe7cd5-a9a7-43dc-b388-330cda8891c0)


We are the same buddy. 
idk what the h is the problem here.
I've already sent a ticket to Microsoft support. 
Their response was, Do this, do that for the GitHub education application," which I've already done.
I've already told the guy that I've already been acknowledged or verified by GitHub Education, and only in the academic verification preprocessing am I not
 

i

**Reply by SH20RAJ** [reply]
Ya , I gave up and purchased digital ocean already, I also have AWS account but it's so costly....

**Reply by jericho1050** [reply]
> Ya , I gave up and purchased digital ocean already, I also have AWS account but it's so costly....

I'm still going to give it a chance because it would be a waste to not take advantage of that benefit.
BTW, how much did it cost you in the digital ocean?
AWS Docs kind of sht. tried it and asked for a dollar, then when you want to stop running the services, there's no way to conveniently stop everything. So what I did was delete my account LMAO.
